src/base/utils/sys.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Bittorrent Client using Qt and libtorrent.
+ * Copyright (C) 2018  Vladimir Golovnev <glassez@yandex.ru>
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * In addition, as a special exception, the copyright holders give permission to
+ * link this program with the OpenSSL project's "OpenSSL" library (or with
+ * modified versions of it that use the same license as the "OpenSSL" library),
+ * and distribute the linked executables. You must obey the GNU General Public
+ * License in all respects for all of the code used other than "OpenSSL".  If you
+ * modify file(s), you may extend this exception to your version of the file(s),
+ * but you are not obligated to do so. If you do not wish to do so, delete this
+ * exception statement from your version.
+ */
+
+#pragma once
+
+#include <QtGlobal>
+
+namespace Utils
+{
+    namespace Sys
```

Comment:
maybe "FileAssociation"

Reply:
>What does 'Sys' mean here? If it is 'system'

Yes, I meant "System" (implying "Operating System").

>the term is wrong, because you are referring to an entity outside of the current one.

That sounds confusing. What entity I am referring to? What "current one" means in this context?

Reply:
You are referring to the operating system, which is outside of the qBittorrent utillity functions.

Reply:
>operating system, which is outside of the qBittorrent utillity functions.

What is it? Can you describe the problem objectively? Unfortunately, I can't read your mind...

Reply:
There is no need to read my mind, let's simply read together the words you wrote. So, we are "inside" qBt project and read "namespace Utils". Fine, this starts an entity that contains utility functions. Now we are "inside" that function group and the next words we read is "namespace Sys(tem)". These must be creating a container for a group of related or organised  entities (functions) sharing a common target or purpose (that follows from the meaning of the word 'system'). Doesn't correspond to what you meant, right?

Reply:
Yes, I want to move here some (operating) system-dependent utils.

Reply:
"OperatingSystem" or "OS" would be a suitable name then.

Reply:
does it make sense to move it under src/gui directory?

Reply:
>"OperatingSystem" or "OS" would be a suitable name then.

Why "Sys" is not an abbreviation for "OperatingSystem" (from your POV)?

Reply:
Because it is an obvious abbreviation of the word "System".

Reply:
>Because it is an obvious abbreviation of the word "System".

Of course. But what another "System" (except operating system) can you associate with it in discussed context?

Reply:
See the comment just above above (https://github.com/qbittorrent/qBittorrent/pull/9057#discussion_r193324129). If that does not help, open a dictionary of your choice and read that the word "system" has a meaning on its own.